c2corg_ui/views/xreport.py

```diff
@@ -20,6 +22,29 @@ def redirect_to_full_url(self):
 
     @view_config(route_name='xreports_view')
     def detail(self):
```

Comment:
This method behaviour seems to be the same than for the profile data: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/views/profile.py#L21-L71

but I notice significant differences, is there a reason for this?

For instance profile method has a ``load_data`` function:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/views/profile.py#L64-L71
```
        def load_data(old_api_cache_key=None):
            not_modified, api_cache_key, document_and_locale = \
                self._get_profile_info(id, lang, old_api_cache_key)
            return not_modified, api_cache_key, document_and_locale

        return self._get_or_create(
            (id, lang), cache_document_detail, load_data, render_page,
            self._get_cache_key)
```

whereas the xreport detail function finishes with
```
return self._get_or_create_detail(id, lang, render_page)
```